apps/06Checkers.js

```diff
@@ -5,26 +5,35 @@ var prompt = require('prompt');
 prompt.start();
 
 
-function Checker() {
+function Checker(color) {
     // Your code here
+    if (color == 'white') {
```

Comment:
It wouldn't be a bad idea to scrub input just in case before checking the color:

`
color = color.toLowerCase();
`